README.md

```diff
@@ -44,6 +44,10 @@ providers:
       port: 8081
       # The api key that grans access (required, example is using an env var)
       api_key: env/POWERDNS_API_KEY
+      # The URL scheme (optional, default http)
+      # scheme: https
+      # Check SSL certificate (optional, default True)
+      # ssl_verify: False
```

Comment:
Generally the value we put in the doc is the default. Also lowercased it as I believe that's the default/standard with YAML.

```suggestion
      # ssl_verify: true
```